pkg/notification/flags.go

```diff
@@ -0,0 +1,58 @@
+package notification
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/trivy/pkg/flag"
+)
+
+// getUsedFlags extracts the fixed flags from the provided args and returns them as a slice of strings.
+func getUsedFlags(cliOptions *flag.Options) string {
```

Comment:
Just as an idea:
we can move this logic to `flag` package.
And keep in `Options` only used flags.
WDYT?

Then we don't need to add IsExplicitlySet function to `Flagger`.
https://github.com/aquasecurity/trivy/blob/92bad18b1e1824c56f2cd9ccfb02b6d2f7498ad3/pkg/flag/options.go#L228